### PR TITLE
Add period and fix grammar

### PR DESCRIPTION
**Pull request by lukeledvina**
#### Summary

Added "and" instead of a comma between two options. Added a period at the end of the same sentence.
#### Motivation

Helps resolve confusion, easier to read.
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/css_functions/index.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ These functions may be used wherever an {{cssxref("&lt;image&gt;")}} is valid as
 - {{cssxref("gradient/conic-gradient()", "conic-gradient()")}}
   - : Conic gradients transition colors progressively around a circle.
 - {{cssxref("image/image()","image()")}} {{Experimental_Inline}}
-  - : Defines an {{cssxref("&lt;image&gt;")}} in a similar fashion to the {{cssxref("url()")}} function, but with added functionality including specifying the image's directionality, specifying fallback images for when the preferred image is not supported
+  - : Defines an {{cssxref("&lt;image&gt;")}} in a similar fashion to the {{cssxref("url()")}} function, but with added functionality including specifying the image's directionality and specifying fallback images for when the preferred image is not supported.
```
Comment (teoli2003) [inline files/en-us/web/css/css_functions/index.md:184]:
```suggestion
  - : Defines an {{cssxref("&lt;image&gt;")}} in a similar fashion to the {{cssxref("url()")}} function, but with added functionality including specifying the image's directionality and fallback images for when the preferred image is not supported.
```

Comment (teoli2003) [reply files/en-us/web/css/css_functions/index.md:184]: I would even remove the second specifying. What do you think?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
With this tiny change done, I think it will be good to go.

File: files/en-us/web/css/css_functions/index.md (hunk 1 above)
Comment (lukeledvina) [reply files/en-us/web/css/css_functions/index.md]: Good catch, that works even better.

**Review by teoli2003 — APPROVED** [review]
Thanks a lot!

**Pull request merged**